docs/data/date-pickers/custom-field/behavior-masked-text-field/MaskedMaterialTextField.tsx

```diff
@@ -2,7 +2,6 @@ import * as React from 'react';
 import dayjs, { Dayjs } from 'dayjs';
```

Comment:
- Unify the wording by removing `ValueStr` everywhere in favor of `InputValue`
- Don't check if `onChange` exists since it always does when used inside a picker
- Remove `defaultValue` prop and assume `value` is always defined (which is the case when the field is used inside a picker)